bank_integration/scripts/payment_entry.js

```
@@ -2,13 +2,13 @@
 
 frappe.ui.form.on('Payment Entry', {
-    setup: function(frm) {
-        frappe.realtime.on("eval_js", function(message){
-            eval(message);
-        })
-    },
-
-	onload: function(frm) {
+    setup(frm) {
+		frappe.realtime.on("eval_js", function(message){
+			eval(message);
```

Biome (2.4.6)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)